etl/types/trasnformer_impl.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/block-explorer/blob/master/LICENSE.md.
+
+package types
+
+import (
+	"github.com/insolar/block-explorer/utils"
+	"github.com/insolar/insolar/insolar"
+	ins_record "github.com/insolar/insolar/insolar/record"
+	"github.com/insolar/insolar/ledger/heavy/exporter"
+	"github.com/pkg/errors"
+)
+
+type Transform interface {
+	Transform() (*JetDrop, error)
+}
+
+const (
+	// delta between pulses
+	pulseDelta uint16 = 10
+)
+
+// Transform transforms thr row JetDrops to canonical JetDrops
+func (jd *PlatformJetDrops) Transform() ([]*JetDrop, error) {
+	// if no records per pulse
+	if len(jd.Records) == 0 {
+		return make([]*JetDrop, 0), nil
+	}
+
+	pulseData, err := getPulseData(jd.Records[0])
+	if err != nil {
+		return nil, err
+	}
+
+	m, err := getRecords(jd.Records)
+	if err != nil {
+		return nil, err
+	}
+
+	result := make([]*JetDrop, 0)
+	for jetID, records := range m {
+		sections := make([]Section, 0)
+		prefix := jetID.Prefix()
+		tmpSection := MainSection{
+			Start: DropStart{
+				PulseData:           pulseData,
+				JetDropPrefix:       prefix,
+				JetDropPrefixLength: uint(len(prefix)),
+			},
+			DropContinue: DropContinue{},
+			Records:      records,
+		}
+		sections = append(sections, tmpSection)
+
+		hash, err := hash(records)
+		if err != nil {
+			return nil, errors.Wrapf(err, "cannot calculate JetDrop hash")
+		}
+		localJetDrop := &JetDrop{
+			Sections: sections,
+			RawData:  hash,
+		}
+		result = append(result, localJetDrop)
+	}
+
+	return result, nil
+}
+
+// hash calculate hash from record's hash
+func hash(r []Record) ([]byte, error) {
+	l := len(r)
+	data := make([][]byte, l)
+	for i, record := range r {
+		data[i] = record.Hash
+	}
+	return utils.Hash(data)
+}
+
+func getPulseData(rec *exporter.Record) (Pulse, error) {
+	r := rec.GetRecord()
+	pulse := r.ID.Pulse()
+	time, err := pulse.AsApproximateTime()
+	if err != nil {
+		return Pulse{}, errors.Wrapf(err, "could not get pulse ApproximateTime")
+	}
+	return Pulse{
+		PulseNo:        int(pulse.AsUint32()),
+		EpochPulseNo:   int(pulse.AsEpoch()),
+		PulseTimestamp: time,
+		NextPulseDelta: int(pulse.Next(pulseDelta)),
```

Comment:
`pulse.Next` return next pulse number, not `NextPulseDelta`. I think for now we can just set `pulseDelta` in this field